docassemble/assemblylinewizard/interview_generator.py

```diff
@@ -529,6 +575,7 @@ def __init__(self, var_name: str, var_type: str, fields: List[DAField]):
         self.attribute_map[plain_att] = (disp_att, settable_att)
 
   def revisit_page(self) -> str:
+    # TODO: deprecated
```

Comment:
If we aren't using it, I'd just delete it.

Reply:
https://github.com/SuffolkLITLab/docassemble-assemblylinewizard/commit/a63d9def3729106e9158cbc1ae605f483ce89fc7